tests/test_source.py

```diff
@@ -119,3 +119,15 @@ def test_THREDDSMergedSource_xarray_kwargs(THREDDSMergedSource_cat_short_url, dr
         assert 'units' not in ds.time.attrs
     else:
         assert 'units' in ds.time.attrs
+
+
+def test_concat_dim():
+    url = "simplecache::https://www.ncei.noaa.gov/thredds/catalog/model-gefs-003/202008/20200831/catalog.xml"
+    cat = intake.open_thredds_merged(url, ['NCEP gens-a Grid 3 Member-Forecast 16-372 for 2020-08-31*'], driver='netcdf', xarray_kwargs=dict(
+        engine="cfgrib",
```

Comment:
I will switch to another netcdf file once I get this running. Although it’s also a nice test to see that it works for grib also

Reply:
Switch testing file and remove cfgrib or keep testing cfgrib file and new dependency? @andersy005

Reply:
just changed path, therefore outdated, but question still standing.

Reply:
> Switch testing file and remove cfgrib or keep testing cfgrib file and new dependency? @andersy005

👍🏽 for adding `cfgrib` as a testing dependency